From d2aa3590b210bc6870f17d3e6321f246303d9c50 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 20 Apr 2012 18:16:24 +0800
Subject: ENGR00179636-02 - FEC : Enet RX FIFO overruns issue.

- Increase RX BD size to 384 entrys from 16 entrys, which
  can reduce the overruns number in busy system.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 drivers/net/fec.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/fec.c b/drivers/net/fec.c
index a93ac31..0fe1175 100755
--- a/drivers/net/fec.c
+++ b/drivers/net/fec.c
@@ -119,7 +119,7 @@ MODULE_PARM_DESC(macaddr, "FEC Ethernet MAC address");
  * We don't need to allocate pages for the transmitter.  We just use
  * the skbuffer directly.
  */
-#define FEC_ENET_RX_PAGES	8
+#define FEC_ENET_RX_PAGES	192
 #define FEC_ENET_RX_FRSIZE	2048
 #define FEC_ENET_RX_FRPPG	(PAGE_SIZE / FEC_ENET_RX_FRSIZE)
 #define RX_RING_SIZE		(FEC_ENET_RX_FRPPG * FEC_ENET_RX_PAGES)
-- 
1.7.9.5

